custom_components/opnsense/coordinator.py

```
@@ -71,225 +72,231 @@
+                    if self._mismatched_count == 3:
+                        ir.async_create_issue(
+                            hass=self.hass,
+                            domain=DOMAIN,
+                            issue_id=f"{self._device_unique_id}_device_id_mismatched",
+                            is_fixable=False,
+                            is_persistent=False,
+                            severity=ir.IssueSeverity.ERROR,
+                            translation_key="device_id_mismatched",
+                        )
+                        _LOGGER.error(
+                            "OPNsense Device ID has changed which indicates new or changed hardware. "
```

Pylint (3.3.7)

[convention] Line too long (102/100)

(C0301)